.github/workflows/examples_build-host-test.yml

```diff
@@ -51,3 +64,38 @@ jobs:
           python ./ci/build_apps.py examples/mqtt -l -t linux
           timeout 5 ./examples/mqtt/build_linux_default/esp_mqtt_demo.elf | tee test.log  || true
           grep 'MQTT_EVENT_DATA' test.log
+
+  run_on_target:
+    # Skip running on forks since it won't have access to secrets
+    if: |
+      github.repository == 'espressif/esp-protocols' &&
+      ( contains(github.event.pull_request.labels.*.name, 'examples') || github.event_name == 'push' )
+    name: Slip example target test
+    needs: build_all_examples
+    strategy:
+      matrix:
+        idf_ver: ["release-v5.3", "latest"]
```

Comment:
run on v5.4, too
```suggestion
        idf_ver: ["release-v5.3", "release-v5.4", "latest"]
```

Reply:
* added `v5.4` to build jobs
* replaced `v5.3` -> `v5.4` in run jobs.
(so we're building on all versions, but running only on latest)